network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,78 @@ These release notes track the development of the Network Observability Operator
+With this fix, setting the `spec.console.register` value to false does not impact the console plugin registration or registration removal, meaning that the plugin can be safely registered by other means. (link:https://issues.redhat.com/browse/NETOBSERV-1134[*NETOBSERV-1134*])
+* Previously, using the default metrics settings, the *NetObserv/Health* dashboard was showing an empty graph named "Flows Overhead". You could still get this metric by removing "namespaces-flows" and "namespaces" from the list of ignored tags. Now, using the default metrics settings, this metric is visible. (link:https://issues.redhat.com/browse/NETOBSERV-1351[*NETOBSERV-1351*])
+* Previously, with a specific cluster configuration, the node on which the eBPF Agent was running was not resolving, which resulted in cascading consequences that ended up in failure to provide some of the traffic metrics. Now the eBPF agent's node IP is safely provided by the Operator, inferred from the pod status, which restores those missing metrics. (link:https://issues.redhat.com/browse/NETOBSERV-1430[*NETOBSERV-1430*])
+* Previously, when the Loki error 'Input size too long' occurred, users did not have much information about it; they had to search the documentation or the knowledge database.
+Now, some help is directly displayed next to the error with a direct link for more guidance. (link:https://issues.redhat.com/browse/NETOBSERV-1464[*NETOBSERV-1464*])
+* Previously, the console plugin read timeout was forced to 30s.
+Now, with the `FlowCollector` `v1beta2` API update, you can configure `readTimeout` in the `loki` section to update this value accordingly with loki operator `queryTimeout` limit. (link:https://issues.redhat.com/browse/NETOBSERV-1443[*NETOBSERV-1443*])
+* Previously, the Operator bundle did not advertise some supported features as expected by CSV annotations, such as "features.operators.openshift.io/..."
+Now these annotations are set in the CSV as expected. (link:https://issues.redhat.com/browse/NETOBSERV-1305[*NETOBSERV-1305*])
+* Previously, the `FlowCollector` status could sometimes oscillate between "DeploymentInProgress" and "Ready" states during reconciliation.
+Now, actual readiness is correctly detected, and the status only becomes "Ready" when all underlying components are fully ready.(link:https://issues.redhat.com/browse/NETOBSERV-1293[NETOBSERV-1293])
+
+[id="network-observability-operator-1.5.0-known-issue"]
+=== Known issues
+* When trying to access the web console, cache issues on OCP 4.14.10 prevent access to the *Observe* view. The web console shows the error message: `Failed to get a valid plugin manifest from /api/plugins/monitoring-plugin/`. The recommended workaround is to update the cluster to the latest minor version. If this does not work, you need to apply the workarounds described in this link:https://access.redhat.com/solutions/7052408[Red Hat Knowledgebase article].(link:https://issues.redhat.com/browse/NETOBSERV-1493[*NETOBSERV-1493*])
+
```

Comment:
@skrthomas : should we add Performance bug as a known issue? cc @openshift/netobserv-team

Reply:
@memodi I think in the past releases, we've decided to leave general performance bugs out of the known issues. The only time I think we noted it was when there was a specific configuration that was related to performance. That being said, I defer to SMEs about this.

Reply:
@skrthomas - no worries. We had discussion in QE sync, I am fine with it being left out of RN and we can keep working it internally.